modules/rosa-cluster-autoscaler-settings.adoc

```diff
@@ -0,0 +1,113 @@
+//
+// * rosa_cluster_admin/rosa-cluster-autoscaling.adoc
+:_content-type: REFERENCE
+[id="rosa-cluster-autoscale-settings_{context}"]
+= Cluster autoscaling settings
+
+The tables explains all the editable autoscaling settings when using cluster autoscaling with OpenShift Cluster Manager.
+
+== Balancing behavior
+
+[cols="4",options="header"]
+|===
+|Setting
+|Description
+|Type or Range
+|Default
+
+|`balanceSimilarNodeGroups`
+|BalanceSimilarNodeGroups enables or disables the `--balance-similar-node-groups` cluster-autoscaler feature. This feature automatically identifies node groups with the same instance type and the same set of labels and tries to keep the respective sizes of those node groups balanced.
+|`boolean`
+|`false`
+
+|`logVerbosity`
+|Sets the autoscaler log level. Default value is 1, level 4 is recommended for DEBUGGING and level 6 enables almost everything. This option has priority over log level set by the `CLUSTER_AUTOSCALER_VERBOSITY` environment variable.
+|`integer`
+|1
+
+|`maxPodGracePeriod`
+|Gives pods graceful termination time before scaling down.
+|`integer`
+|600
+
+|`skipNodesWithLocalStorage`
+|Enables or disables `--skip-nodes-with-local-storage` CA feature flag. If `true`, cluster autoscaler never deletes nodes with pods with local storage, e.g. EmptyDir or HostPath.
+|`boolean`
+|`true`
+|===
+
+== Resource limits
+
+[cols="4",options="header"]
+|===
+|Setting
+|Description
+|Type or Range
+|Default
+
+|`maxNodesTotal`
+|Maximum number of nodes in all node groups. Includes all nodes, not just automatically scaled nodes. Cluster autoscaler does not grow the cluster beyond this number.
+|`integer`
+|0
+
+|`cores.min`
+|Minimum number of cores in cluster. Cluster autoscaler does not scale the cluster beyond these numbers.
+|`object`
+|1
+
+|`cores.max`
+|Maximum number of cores in cluster. Cluster autoscaler does not scale the cluster beyond these numbers.
+|`object`
+|320000
+
+|`memory.min`
+|Minimum number of gigabytes of memory in cluster. Cluster autoscaler does not scale the cluster beyond these numbers.
+|`object`
+|1
+
+|`memory.max`
+|Maximum number of gigabytes of memory in cluster. Cluster autoscaler does not scale the cluster beyond these numbers.
+|`object`
+|6400000
+|===
+
+== Scale down configuration
+
+[cols="4",options="header"]
+|===
+|Setting
+|Description
+|Type or Range
+|Default
+
+|`enable`
+|Should CA scale down the cluster.
+|`boolean`
+|`true`
+
+|`scaleDown.unneededTime`
+|How long a node should be unneeded before it is eligible for scale down.
+|`string`
+|10m
+
+|`scaleDown.utilizationThreshold`
+|Node utilization level, defined as sum of requested resources divided by capacity, below which a node can be considered for scale down.
+|`string`
+|0.5
+
+|`scaleDown.delayAfterAdd`
+|How long after scale up that scale down evaluation resumes.
+|`string`
+|10m
+
+|`scaleDown.delayAfterDelete`
+|How long after node deletion that scale down evaluation resumes, defaults to `scan-interval`.
```

Comment:
ATM, we're showing `scan-interval` in the dialog, and we submitting an empty string to the API.

Can you confirm that sending '0s' is equivalent to sending an empty string? It would simplify the code a bit a would keep this setting same as all the others.

Reply:
@celdrake yes, it seems like cluster-autoscaler doesn't choke on ``--scale-down-delay-after-delete="0s"`` and proceed to be running as intended. So you can use it as the value

Reply:
Removed the default value of "scan-interval" and instead show it as "0s". We send this value as well when the default is selected.